sdk/tables/Azure.Data.Tables/tests/TableClientLiveTests.cs

```diff
@@ -300,7 +300,7 @@ public async Task EntityMergeDoesPartialPropertyUpdates()
 
             // Do not provide an ETag to update unconditionally.
```

Comment:
This comment probably could be removed now since `UpsertMergeAsync` doesn't accept an ETag value.